src/Valit/Rules/ValitRule.cs

```diff
@@ -77,5 +85,14 @@ public IValitResult Validate(TObject @object)
 
             return !hasAllConditionsFulfilled || isSatisfied ? ValitResult.Success : ValitResult.Fail(_errors.ToArray());
         }
+
+        IEnumerable<IValitRule<TObject>> IValitRule<TObject>.GetEnsureRules(TObject @object)
+        {
+            @object.ThrowIfNull();
+            if (_ruleFunc != null)
```

Comment:
It doens't change anything, but I think in the other places we're using an `if` we're surrounding it with braces.
Personally I would also check if `_ruleFunc` is `null` instead of not `null`, contradictory `if`s are harder to read imo, @GooRiOn thoughts?